tests/unittests/core/evc/conftest.py

```diff
@@ -10,46 +10,46 @@
 @pytest.fixture
 def parent_config():
     """Generate a new experiment configuration"""
-    return dict(_id="test", name="test", metadata={"user": "corneauf"}, version=1)
+    return dict(_id=1234, name="test", metadata={"user": "corneauf"}, version=1)
 
 
 @pytest.fixture
 def child_config(parent_config):
     """Generate a new experiment configuration"""
     config = copy.deepcopy(parent_config)
-    config["_id"] = "test2"
-    config["refers"] = {"parent_id": "test"}
+    config["_id"] = 1235
+    config["refers"] = {"parent_id": 1234}
     config["version"] = 2
 
     return config
 
 
 @pytest.fixture
-def exp_no_child_conflict(create_db_instance, parent_config):
+def exp_no_child_conflict(pdatabase, parent_config):
     """Generate an experiment name conflict"""
-    create_db_instance.write("experiments", parent_config)
+    pdatabase.write("experiments", parent_config)
```

Comment:
```suggestion
    storage.create_experiment(parent_config)
```

Reply:
And the same for all the rest below. :grimacing:

Reply:
Okay, I'll run the good old search and replace.